kDrive/AppDelegate.swift

```diff
@@ -105,12 +105,21 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // swiftlint:disable force_try
         Task {
-            try! await Task.sleep(nanoseconds:5_000_000_000)
-            print("coucou")
+            try! await Task.sleep(nanoseconds: 5_000_000_000)
```

Comment:
Please leave this test code here, it will be removed in a later PR, this is only about the 2 new views.